code/modules/mob/living/carbon/human/death.dm

```diff
@@ -49,25 +49,20 @@
 /mob/living/carbon/human/dust()
 	if(!death(TRUE) && stat != DEAD)
 		return FALSE
-	notransform = 1
-	canmove = 0
-	icon = null
-	invisibility = 101
+	notransform = TRUE
+	canmove = FALSE
 	dust_animation()
-	QDEL_IN(src, 15)
+	QDEL_IN(src, 20)
 	return TRUE
 
 /mob/living/carbon/human/dust_animation()
-	var/atom/movable/overlay/animation = null
+	// Animate them being dusted out of existence
+	var/obj/effect/snap_rt/dust_effect = new(loc, UID())
+	filters += filter(type="displace", size=256, render_source="*snap[UID()]")
+	animate(src, alpha=0, time=20, easing=(EASE_IN | SINE_EASING))
```

Comment:
Question regarding this code btw. This is what actually does the effect right? The `/obj/effect/snap_rt` object itself is just a holder with no behaviour right?
If so wouldn't it make more sense to move this logic to the effect itself?

Reply:
Probably. Ill try that tomorrow.

Reply:
```dm
	var/mutable_appearance/MA = mutable_appearance('icons/effects/filters.dmi', "snap3", ABOVE_ALL_MOB_LAYER, FLOOR_PLANE)
	MA.render_target = "dusting_[UID()]"
	filters += filter(type="displace", size=256, render_source="dusting_[UID()]")
```

Does not work

Reply:
I honestly don't exactly know how `render_target` works for byond.
But I meant to say to move the logic itself to `/obj/effect/snap_rt` or a global proc which does the animation for you. Currently, you can spawn `/obj/effect/snap_rt` with the proper params and it does nothing on its own.

Reply:
Have you tried with the asterisk? Render targets that start with it are handled differently.

Reply:
* dont help it